app/decorators/calendar_event_decorator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CalendarEventDecorator < SimpleDelegator
+  def active_now?
```

Comment:
Bu metot `CalendarEvent` sınıfına ait bir metot olmalı. Herhangi bir takvim etkinliğinin aktif olup olmadığını direk kendi üzerinden öğrenebilmeliyim. Üstelik başka controller, delegator vs. içerisinde de kullanıldığı için direk nesnenin kendisinden öğrenilebilecek bir logic.